kanidmd/lib/src/idm/server.rs

```diff
@@ -620,7 +620,7 @@ pub trait IdmServerTransaction<'a> {
             security_info!("The token grace window is in effect. Assuming valid.");
         };
 
-        Account::try_from_entry_no_groups(entry.as_ref()).map(Some)
```

Comment:
Why not just use `Account::try_from_entry_ro` here?

Reply:
Because we do not need the group information in every case (e.g the token introspection endpoint) so this saves a few DB queries